utils/lib/rr_graph/tests/test_graph.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python3
-# Run `python3 -m unittest utils.lib.rr_graph.tests.test_graph`
+# Run `python3 -m unittest lib.rr_graph.tests.test_graph`
 import unittest
 
-from .. import graph, P, Size
```

Comment:
Why have you changed these to absolute imports?

Reply:
pytest failed on these imports. I think this is because of how the modules are laid out. I haven't dug into it but I believe we need to separate some of our code some and make sure we take care of some details around module best practices.

Reply:
I think we want lib/rr_graph to be its own library and then we want to add `utils/lib` to python path and then import this as `import rr_graph`or `pytest rr_graph` style thing?